Automattic/simplenote-macos
# Fixing Checklist Asset

### Fix
In this PR we're rotating the Checklist Asset.

No, I'm still not sure how would this break... but... Xcode 11 has something to do, for granted.

Closes #401

@aerych Eric, may I bug you yet again?
Thank you!!!

### Test
1. Log into your account
2. Insert a checklist (`Format` > `Insert Checklist`)
3. Click over the new checklist asset

- [x] Verify the Checked State looks great!


### Release
These changes do not require release notes.
